contracts/AuctionBase.sol

```diff
@@ -1,51 +1,37 @@
 pragma solidity ^0.4.18;
 
-import "./ERC721/ERC721.sol";
-import "./Pausable.sol";
+import "./erc721/ERC721.sol";
+import "./utils/Pausable.sol";
 
 /// @title AuctionBase
 /// @dev Contains models, variables, and internal methods for the auction.
 contract AuctionBase is Pausable {
 
   struct Auction {
     // static
-    // NFT address
-    address nftAddress;
-    // ID of the nft
-    uint256 tokenId;
-    // Current owner of NFT
-    address seller;
-    // Minimum bid increment (in Wei)
-    uint128 bidIncrement;
-    // Duration (in seconds) of auction
-    uint64 duration;
-    // Time when auction started
-    // NOTE: 0 if this auction has been concluded
-    uint64 startedAt;
+    address nftAddress; // NFT address
+    uint256 tokenId; // ID of the nft
+    address seller; // Current owner of NFT
+    uint128 bidIncrement; // Minimum bid increment (in Wei)
+    uint64 duration; // Duration (in seconds) of auction
+    uint64 startedAt; // Time when auction started (0 if auction is concluded)
 
     // state
-    // Mapping of addresses to balance available to withdraw
-    mapping (address => uint256) allowed;
-    // Current highest bid
-    uint256 highestBid;
-    // Address of current highest bidder
-    address highestBidder;
+    mapping (address => uint256) allowed; // Mapping of addresses to balance to withdraw
+    uint256 highestBid; // Current highest bid
+    address highestBidder; // Address of current highest bidder
+    bool cancelled; // Flag for cancelled auctions
   }
 
   // Map from token ID to their corresponding auction ID.
   mapping (address => mapping(uint256 => uint256)) nftToTokenIdToAuctionId;
-
-  // Array of auctions
   Auction[] public auctions;
 
   event AuctionCreated(uint256 id, address nftAddress, uint256 tokenId);
   event AuctionSuccessful(uint256 id, address nftAddress, uint256 tokenId);
   event AuctionCancelled(uint256 id, address nftAddress, uint256 tokenId);
   event BidCreated(uint256 id, address nftAddress, uint256 tokenId, address bidder, uint256 bid);
 
-  /// @dev DON'T give me your money.
-  function() external {}
-
```

Comment:
This is the fallback function, we should probably keep it and propabably add ```revert()``` in the function body. It's the function that gets call if someone tries to send ETH using sendTransaction to the smart contract address.  Some people are dumb and if they do that, they will lose their ETH and we get free ETH, but not cool to keep it.